frontend/src/components/Election/Results/STARResultSummaryWidget.tsx

```diff
@@ -22,29 +22,37 @@ const renderCustomizedLabel = ({ cx, cy, midAngle, innerRadius, outerRadius, per
     );
 };
 
-const STARResultSummaryWidget = ({ results, rounds }) => {
-    const histData = results.summaryData.candidates.map((c, n) => ({
-        name: c.name,
-        votes: results.summaryData.totalScores[n].score,
-        // vvvv HACK to get the bars to fill the whole width, this is useful if we want to test the graph padding
-        votesBig: results.summaryData.totalScores[n].score*10000 
-    }));
+const STARResultSummaryWidget = ({ results, roundIndex }) => {
+    const prevWinners = results.roundResults
+        .filter((_, i) => i < roundIndex)
+        .map(round => round.winners)
+        .flat(1)
+        .map(winner => winner.index);
+
+    const histData = results.summaryData.candidates
+        .map((c, n) => ({
+            name: c.name,
+            votes: results.summaryData.totalScores[n].score,
+            // vvvv HACK to get the bars to fill the whole width, this is useful if we want to test the graph padding
+            votesBig: results.summaryData.totalScores[n].score*10000 
+        }))
+        .filter((_, i) => !prevWinners.includes(i));
```

Comment:
I'm wondering if instead of filtering out candidates we should just mark previous winners or grey out their scores.

Reply:
We discussed this in a meeting, and decided to leave it as is. Greying out candidates could create confusion at what the stars mean in the scoring round